internal/pkg/cli/svc_package.go

```diff
@@ -333,6 +337,7 @@ func (o *packageSvcOpts) getWorkloadStack(generator workloadStackGenerator) (*cf
 			EnvFileARN:         uploadOut.EnvFileARN,
 			AddonsURL:          uploadOut.AddonsURL,
 			CustomResourceURLs: uploadOut.CustomResourceURLs,
+			EnvVersion:         envVersion,
```

Comment:
I do feel like grabbing & passing `envVersion` is better placed in `cli/deploy`. But that involves slightly more work to refactor and I think it's okay to do this after this patch release.

Reply:
i moved it to `cli/deploy`!